server/jetstream_cluster_test.go

```diff
@@ -10676,20 +10676,33 @@ func TestJetStreamClusterStreamReplicaUpdateFunctionCheck(t *testing.T) {
 	checkSubsPending(t, sub, 3*numMsgs)
 
 	// Make sure cluster replicas are current.
-	for _, r := range si.Cluster.Replicas {
-		if !r.Current {
-			t.Fatalf("Expected replica to be current: %+v", r)
+	checkFor(t, 5*time.Second, 100*time.Millisecond, func() error {
+		si, err = js.StreamInfo("TEST")
+		require_NoError(t, err)
```

Comment:
You can't use that here... you would need: `if err != nil { return err }`...

Reply:
We should always be able to get stream info, so that is fatal to me for the test if we get an error.